cmake/kokkos_test_cxx_std.cmake

```diff
@@ -14,12 +14,12 @@ FUNCTION(kokkos_set_cxx_standard_feature standard)
     ENDIF()
   ELSEIF(CMAKE_CXX_EXTENSIONS)
     IF(KOKKOS_DONT_ALLOW_EXTENSIONS)
-      MESSAGE(FATAL_ERROR "The chosen configuration does not support CXX extensions flags: ${KOKKOS_DONT_ALLOW_EXTENSIONS}. Must set CMAKE_CXX_EXTENSIONS=OFF to continue")
+      MESSAGE(FATAL_ERROR "The chosen configuration does not support CXX extensions flags: ${KOKKOS_DONT_ALLOW_EXTENSIONS}. Must set CMAKE_CXX_EXTENSIONS=OFF to continue") 
     ELSE()
       GLOBAL_SET(KOKKOS_USE_CXX_EXTENSIONS ON)
     ENDIF()
   ELSE()
-    #For trilinos, we need to make sure downstream projects
+    #For trilinos, we need to make sure downstream projects 
```

Comment:
Do not add trailing whitespaces

Reply:
This is technically not adding a whitespace, this commit reverts a change which should not have happened. The whitespace is probably present in develop.

Reply:
Yeah just checked the whitespace is present in develop. This file is restored to the exact version in develop.